README.md

```diff
@@ -1,11 +1,59 @@
 # Museum of Dreams Website Project
 
-This website is hosted at http://museumofdreams.eu-west-2.elasticbeanstalk.com/
+The production website is hosted at http://museumofdreamworlds.eu-west-2.elasticbeanstalk.com/
```

Comment:
We need to sort out https for this right? Let's create a separate issue for that